zenoh/src/net/runtime/adminspace.rs

```diff
@@ -471,6 +489,40 @@ fn router_data(context: &AdminContext, query: Query) {
     }
 }
 
+fn router_metrics(context: &AdminContext, query: Query) {
+    let reply_key: OwnedKeyExpr = format!("@/router/{}/metrics", context.zid_str)
```

Comment:
Should we call it @/router/{zid_str}/stats instead of @/router/{zid_str}/metrics for consistency with the rest?

Reply:
See above.